### ISSUE TITLE
wiktionary: output improvements

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel#1593]
Extracting the following points from #1443 into a new issue for @HumorBaby so they don't get lost.

- [ ] give sub-definitions like those for [`-able`](https://en.wiktionary.org/wiki/-able) their own sub-number (e.g., 1a., 1b., ...)
- [ ] update output formatting to be smarter about sub-definitions and such
- [ ] reorder parts of speech list so that definitions can be prioritized… it would be a shame if some obscure _circumposition_ (A circumposition is a discontinuous adposition… _um wat?_) definition led to the more common _verb_ definition being truncated.

Lightly edited, nothing more. As far as I know, this is the full remaining extent of improvements @HumorBaby was/is planning to make to the `wiktionary` module for Sopel 7.

**Comment by dgw** [comment]
Given how busy @HumorBaby has been recently, this has a vanishingly small chance of being done for 7.0. It's a straight enhancement, nothing more, so I'm fine waiting on it.

**Comment by dgw** [comment]
Cleared @HumorBaby as assignee, since he's been very busy lately. This is up for grabs if anyone has the itch to work on it.

**Comment by dgw** [comment]
This and #1947 need to be migrated over to sopel-wikimedia. The milestone will be cleared in the mean time, until I can properly triage all the bullet points. x)
